index.js

```diff
@@ -84,6 +84,192 @@ app.post('/add/:quizName/:count', urlencodedParser, async (req, res) => {
     </html>`)    
 })
 
+////
+////
+////These are the /take routes
+
+app.post('/take', urlencodedParser, async(req, res) => {
+    console.log("This makes sure that this is running properly")  // TEMP
+    
+    var quizName = req.body.userName
+    const inCase = 'Sorry, this quiz does not exist.'
+    if (quizName == '') {
+        quizName = inCase
+    }
+
+    var userName = req.body.quizName
+    if (userName == '') {
+        userName = 'unknown user'
+    }
+
+    console.log(quizName) // TEMP
+    
+    var firstQuestion = await db.get(`${quizName}-question-1`)
+    if (firstQuestion == null) {
+        firstQuestion = "Sorry, this thing your on doesn't exist :("
+    }
+    await db.hGet('questionCount', quizName, '0') 
+    
+    if (quizName == inCase) {
+        res.send(`<!DOCTYPE html>
+        <html>
+        <head>
+            <title>Error</title>
+        </head>
+        <body>
+            <h1>${inCase}</h1>
+                <strong>Don't worry; click <em><a href="http://localhost">here</a></em> to return to the homepage.</strong>
+        </body>
+       </html>`)
+    } else {
+
+        res.send(`<!DOCTYPE html>
+        <html>
+        <head>
+            <title>Question 1 in ${quizName}</title>
+        </head>
+        <body>
+            <h1>Question 1 in ${quizName}</h1>
+                <strong>${firstQuestion}</strong>
+                
+                <form action="/take/${quizName}/${userName}/2" method="post">
```

Comment:
Should be continuing on to /1 - submitting the answer for question 1. For example, what if there is no question 2?

In your next handler with `:questionNum`, that's where it should decide if the quiz is done.